variant-article.es6

```diff
@@ -14,6 +15,12 @@ import CallToAction from './cta';
 const config = {
   defaultVariant: 'world-in-main',
   variants: {
+    'world-if': {
+      ArticleHeader: WifHeader,
+      ArticleSubheader: WifSubheader,
+      ArticleBody: WorldIfArticleBody,
```

Comment:
Is there a specific reason we use 2 different prefix "Wif" and "WorldIf"?
If not it will be better using just one.